.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Orb version
```

Comment:
There are three changes in this PR - can you please split it into three PRs please?

- one to add the github issue template scaffolding
- one to factor out the commands into seperate files
- one to add the new functionality